scripts/autogen

```diff
@@ -311,7 +324,60 @@ def gen_aarch64_mulcache_twiddles_twisted():
         yield prepare_root_for_barrett(root)[1]
 
 
-def gen_aarch64_fwd_ntt_zeta_file(dry_run=False):
+def print_hol_light_int_array(g):
+
+    # Format of integer list entries, including `;` separator:
+    # - Positive numbers: &42;
+    # - Negative numbers: -- &42;
+    def format_hol_light_int(n):
+        prefix = ""
+        if n < 0:
+            prefix = "-- "
+            n = -n
+        return f"{prefix}&{n};"
+
+    l = list(map(format_hol_light_int, g))
+    # Remove `;` from end of last entry
+    l[-1] = l[-1][:-1]
+
+    entries_per_line = 8
+    for i in range(0, len(l), entries_per_line):
+        yield "  " + " ".join(l[i : i + entries_per_line])
+
+
+def gen_aarch64_hol_light_zeta_file(dry_run=False):
+    def gen():
+        yield from gen_hol_light_header()
+        yield "(*"
+        yield " * Table of zeta values used in the AArch64 NTTs"
+        yield " * See autogen for details."
+        yield " *)"
+        yield ""
+        yield "let ntt_zetas_layer01234 = define `ntt_zetas_layer01234:int list = ["
+        yield from print_hol_light_int_array(gen_aarch64_fwd_ntt_zetas_layer12345())
```

Comment:
unfortunate that we start counting from 0 in one place and 1 in the other. Is it easy to unify that?

Reply:
Yep! Done